worker/convert/tasks.py

```diff
@@ -20,3 +18,25 @@ def execute_task(self):
 
 
 TifToJpgTask = celery_app.register_task(TifToJpgTask())
+
+
+class ConvertPdfToTxtTask(BaseTask):
```

Comment:
Bitte die Klassen einheitlich benennen, entweder mit "Convert..." oder ohne. Ich präferiere ohne, weil "convert" ja schon der Paketname ist und bei den anderen Tasks die Konvention auch ist, dass der Klassenname dem Teil des Tasknamens nach dem Punkt in Camel Case entspricht.